conf/test_screenpathogen.config

```diff
@@ -31,6 +31,11 @@ params {
     extract_centrifuge_reads      = false
     extract_diamond_reads         = false
 
+    // de novo
+    skip_shortread_denovo         = true
+    skip_longread_denovo          = true
+    min_read_counts               = 18
```

Comment:
Why here is 18 and in the config above is 100? Is it for testing purpose?

Reply:
This contig and `test_taxid.config` only extract reads for a user-defined list of virus taxids. Since not many reads are assigned to them, I set the test value to 18 for testing purposes. I have added the recommended value > 100 to the nextflow schema.

Reply:
Ok!